gui/src/ts/components/kg/node/KgNodeTable.tsx

```diff
@@ -25,66 +15,158 @@ const showListAsColumn = (list: string[]) =>
     </React.Fragment>
   ));
 
+const columns: MUIDataTableColumn[] = [
+  {
+    name: "#",
+    options: {
+      customBodyRender(_, tableMeta) {
+        return (
+          tableMeta.tableState.page * tableMeta.tableState.rowsPerPage +
+          tableMeta.rowIndex +
+          1
+        );
+      },
+    },
+  },
+  {
+    name: "id",
+    options: {
+      display: "false",
+    },
+  },
+  {
+    name: "label",
+    label: "Label",
+    options: {
+      customBodyRender(_, tableMeta) {
+        const nodeRowData = (tableMeta.tableData[
+          tableMeta.rowIndex
+        ] as unknown) as (string | undefined)[];
+
+        return (
+          <KgNodeLink
+            node={{
+              id: nodeRowData[getPropertyColumnIndex("id")]!,
+              label: nodeRowData[getPropertyColumnIndex("label")] || null,
+              pos: nodeRowData[getPropertyColumnIndex("pos")] || null,
+            }}
+          />
+        );
+      },
+    },
+  },
+  {
+    name: "aliases",
+    label: "Aliases",
+    options: {
+      customBodyRender(aliases) {
+        return aliases ? showListAsColumn(aliases as string[]) : null;
+      },
+    },
+  },
+  {
+    name: "datasource",
+    label: "Datasources",
+    options: {
+      customBodyRender(datasources) {
+        return datasources
+          ? (datasources as string).split(",").map((datasource) => (
+              <React.Fragment key={datasource}>
+                <KgDatasourceLink datasource={datasource} />
+                <br />
+              </React.Fragment>
+            ))
+          : null;
+      },
+    },
+  },
+  {
+    name: "other",
+    label: "Other",
+  },
+  {
+    name: "pos",
+    label: "Pos",
+    options: {
+      customBodyRender(pos) {
+        return pos ? showListAsColumn((pos as string).split(",")) : null;
```

Comment:
This split and the datasources above will go away, these need to be parsed into lists on the server side. No need to change your code.

Reply:
Gotcha, will keep an eye out for that